### PR TITLE
ui: improved agenda list

### LINKED ISSUE #322: Improvement on agenda list ui
as per new figma:

### PR DESCRIPTION
**Pull request by deewakar-k**
## Summary by CodeRabbit

- **New Features**
	- Enhanced the Today Meetings component with a new layout for agenda items and improved styling.
	- Introduced a 12-hour time format for meeting times.
	- Added new meeting icons for Google Meet and Zoom to improve visual representation of agenda items.
	- Improved handling of meeting links by deriving them from multiple properties.
	- Expanded the Meeting interface to include comprehensive details about meetings.

- **Bug Fixes**
	- Simplified the display for the "no agenda items" message.

### CODE REVIEW

**Review by immadsahin — CHANGES_REQUESTED** [review]
In case of other meeting urls it shows the same google meet icon, use link icon in case of other meeting links, use zoom icon for zoom meetings.

**Comment by deewakar-k** [comment]
> In case of other meeting urls it shows the same google meet icon, use link icon in case of other meeting links, use zoom icon for zoom meetings.

sure, but was this mentioned anywhere? i couldn't find it.

**Comment by immadsahin** [comment]
> > In case of other meeting urls it shows the same google meet icon, use link icon in case of other meeting links, use zoom icon for zoom meetings.
> 
> sure, but was this mentioned anywhere? i couldn't find it.

yes I forgot to mention this in the issue.

File: apps/frontend/src/components/TodayMeetings.tsx [hunk 1]
```diff
@@ -1,8 +1,56 @@
...
+
+  if (
+    meeting.description?.includes("zoom.us") ||
+    meeting.location?.includes("zoom.us")
+  ) {
+    return "zoom"
+  }
+
+  return "other"
+}
```
[the remark this answers is not in the document]
Comment (sajdakabir) [reply apps/frontend/src/components/TodayMeetings.tsx]: @deewakar-k , please take a look into this suggestion, I really like the suggested coding style.

**Review by immadsahin — CHANGES_REQUESTED** [review]
https://github.com/user-attachments/assets/363eb827-c32a-4053-a931-fc857d42badd

@deewakar-k

File: apps/frontend/src/components/TodayMeetings.tsx [hunk 2]
```diff
@@ -1,8 +1,56 @@
...
+        </div>
+      )
+    default:
+      return (
+        <div className={`mt-[4px]`}>
+          <Link />
+        </div>
+      )
+  }
+}
```
[the remark this answers is not in the document]
Comment (sajdakabir) [reply apps/frontend/src/components/TodayMeetings.tsx]: @deewakar-k , same applies to this one as well.

**Pull request merged**